flink-contrib/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendTest.java

```diff
@@ -442,6 +444,32 @@ public void testDisposeDeletesAllDirectories() throws Exception {
 	}
 
 	@Test
+	public void testGetKeys() throws Exception {
```

Comment:
This test should go into `StateBackendTestBase`. This way, you don't have to repeat it for the Rocks and the Heap backend.